commands/cluster_commands/install

```diff
@@ -188,7 +196,7 @@ echo "installing grapple version: ${GRAPPLE_VERSION}"
 # VERSION=$(prompt_for_input_with_validation "Enter version of the grapple solution framework: " "Semver version is expected. Default value if empty: 0.2.0" "$non_empty_regex" "Version cannot be empty." "0.2.0") || exit $?
 
 # The script then proceeds to dynamically generate values-override.yaml with user inputs
-cat <<EOF > ./values-override.yaml
+cat <<EOF > ./tmp/values-override.yaml
```

Comment:
@zahid-iqbal-alpha 
please use /tmp/ and not ./tmp/
your current code wouldn't work, if there is not ./tmp/ folder.

Reply:
Thanks, I think /tmp/... is the better choice. that should always be there and should be used for that kind of temporary files :-) Also we won't run into permission issues (I hope :-))